### PR TITLE
Geeks3D.FurMark version 1.28.0.0

### PR DESCRIPTION
**Pull request by Rolling2405**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26659)

### CODE REVIEW

File: manifests/g/Geeks3D/FurMark/1.27.0.0/Geeks3D.FurMark.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Geeks3D.FurMark
+PackageVersion: 1.27.0.0
+PackageLocale: en-US
+Publisher: Geeks3D
+PackageName: FurMark
```
Comment (OfficialEsco) [inline manifests/g/Geeks3D/FurMark/1.27.0.0/Geeks3D.FurMark.locale.en-US.yaml:8]:
```suggestion
PackageName: Geeks3D FurMark
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/g/Geeks3D/FurMark/1.27.0.0/Geeks3D.FurMark.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Geeks3D.FurMark
+PackageVersion: 1.27.0.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://us2-dl.techpowerup.com/files/pT8x3kYTnm6leGDUPh_4gg/1630582486/FurMark_1.27.0.0_Setup.exe
```
Comment (ItzLevvie) [inline manifests/g/Geeks3D/FurMark/1.27.0.0/Geeks3D.FurMark.installer.yaml:9]:
```suggestion
  InstallerUrl: https://us2-dl.techpowerup.com/files/FurMark_1.27.0.0_Setup.exe
```

File: manifests/g/Geeks3D/FurMark/1.27.0.0/Geeks3D.FurMark.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Geeks3D.FurMark
+PackageVersion: 1.27.0.0
+PackageLocale: en-US
+Publisher: Geeks3D
+PackageName: FurMark
+License: Copyright © 2007-2021 Geeks3D
+ShortDescription: FurMark CPU/GPU Stress Test
```
Comment (ItzLevvie) [inline manifests/g/Geeks3D/FurMark/1.27.0.0/Geeks3D.FurMark.locale.en-US.yaml:10]:
```suggestion
ShortDescription: FurMark is a lightweight but very intensive graphics card / GPU stress test.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — DISMISSED** [review]

**Comment by jamespik** [comment]
This does not appear to be the original download link from the geeks3d downloads page.

**Comment by Rolling2405** [comment]
> This does not appear to be the original download link from the geeks3d downloads page.

Oops sorry about that! How do I update the link?

**Comment by Rolling2405** [comment]
> Hello @Rolling2405,
> I am sorry to report that the Sha256 Hash does not match the installer. This maybe caused by using a vanity URL rather than a URL directly to the binary.
> 
> Please check the HASH and submit again. Sorry for the inconvenience.

I have checked the hash of the installer and it matches what is already in this submission.

**Comment by OfficialEsco** [comment]
Pretty sure we have to use Techpowerup instead because the official link is a bunch of redirects which sets cookies so it can finally download.
https://geeks3d.com/downloads/2021p/FurMark_1.27.0.0_Setup.exe should be the direct download link but it redirects to https://www.geeks3d.com/dlz/ unless you actually click download on https://www.geeks3d.com/dl/show/672

image

**Comment by Rolling2405** [comment]
> Pretty sure we have to use Techpowerup instead because the official link is a bunch of redirects which sets cookies so it can finally download.
> https://geeks3d.com/downloads/2021p/FurMark_1.27.0.0_Setup.exe should be the direct download link but it redirects to https://www.geeks3d.com/dlz/ unless you actually click download on https://www.geeks3d.com/dl/show/672
> 
> image
> 
> winget download result from https://geeks3d.com/dl/get/672
> 
> ```
> Downloading https://geeks3d.com/dl/get/672
>    120 KB
> Installer hash does not match; to override this check use --force
> ```

I did use Techpowerup originally but @jamespik said

> This does not appear to be the original download link from the geeks3d downloads page.

I think that is why he did not approve it.

**Comment by OfficialEsco** [comment]
I know, that comment was meant for @jamespik which most likely will not come back here since its labeled as `hash mismatch` instead of a backend issue..
They have to add a .validation file for quite a few Downloads (probably for safety purposes) which is why he suggested that change. Which does not work.

**Comment by Rolling2405** [comment]
How do I fix this?

**Comment by OfficialEsco** [comment]
```
URI: https://us2-dl.techpowerup.com/files/pT8x3kYTnm6leGDUPh_4gg/1630582486/FurMark_1.27.0.0_Setup.exe, Validation result: Failed, Http status code: Gone
```
You need to use `https://us2-dl.techpowerup.com/files/FurMark_1.27.0.0_Setup.exe` as Levvie requested.

**Comment by Rolling2405** [comment]
Someone on the core development team please review this pull request

**Comment by ItzLevvie** [comment]
> Someone on the core development team please review this pull request

cc @palenshus  @ChuckFerring

**Review by ItzLevvie — DISMISSED** [review]

**Comment by ryfu-msft** [comment]
@Rolling2405 , sorry for the delay, we missed this PR. 

I just tried the installerURL and it just gave me a 404 not found. Can you verify and update to make sure you have the correct download link

**Comment by KevinLaMS** [comment]
The URL in the manifest is failing for us from the browser and from winget.
image

Is there a better URL to use?

**Comment by Rolling2405** [comment]
> The URL in the manifest is failing for us from the browser and from winget.
> image
> 
> Is there a better URL to use?

I have changed the URL however it is now giving a Error-Hash-Mismatch however I have verified the hash and it is correct

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Comment by Rolling2405** [comment]
@KevinLaMS Can you check this again? I have changed to a working url but for some reason it is saying there is a hash mismatch. However, I have manually checked the hash and it seems correct

**Comment by Rolling2405** [comment]
Screenshot 2021-10-05 003555
Screenshot 2021-10-05 004138

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
The URL that's being used in the package is not a direct link to the file; it's downloading a HTML file rather than an EXE file.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You can set the PR to DRAFT for now until a solution is found; so it's easy to focus on PRs that are ready to be approved using `is:pr is:open draft:false -label:Moderator-Approved -label:Project-File -label:Blocking-Issue -label:Needs-Author-Feedback -label:Validation-Metadata`

**Pull request closed without merging**